public/scripts/author/main.js

```diff
@@ -13,7 +13,17 @@ angular.module('app', [function(){
         console.log('Data recieved: ', data);
         $scope.name = data.breweryDB.name;
         $scope.description = data.breweryDB.description;
-        $scope.label = data.breweryDB.labels.medium;
+
+        // Handle for different types of responses that don't
+        // always have labels images
+        if (data.breweryDB.type === 'beer') {
+          $scope.label = data.breweryDB.labels.medium;
+        } else if (data.breweryDB.type === 'brewery') {
```

Comment:
Now that we know we can get breweries, we should def be thinking of a new user story that involve brewery searches (along with a unique server-side API endpoint for the SPA to hit)